### PR TITLE
Add "How to choose the DATA_SENDING_INTERVAL value" section

### PR DESCRIPTION
**Pull request by Himura2la**
- [x] Review

- [x] Check - https://trello.com/c/Sj8xX03Q/493-cloud4rpi-how-to-choose-the-datasendinginterval-value

- [x] Merge

### CODE REVIEW

**Review by sky3d — APPROVED** [review]

**Review by const314 — APPROVED** [review]

File: docs/faq.md [hunk 1]
```diff
@@ -81,3 +81,14 @@ sudo systemctl disable cloud4rpi.service
...
 sudo systemctl daemon-reload
 ```
+
+
+## How to choose the DATA_SENDING_INTERVAL value?
+
+You should call [device.publish_data()](/api/python/#publish_data) and [device.publish_diag()](/api/python/#publish_diag) functions frequently to update the monitored variables. The recommended update frequency depends on the variable dynamics, for example, atmospheric conditions
+ do not change every minute. This means the delay between updates for a weather station should be one to ten minutes.
+
+If you monitor several variables with different dynamics, the delay between updates should be the shortest of all.
```
Comment (dirk-pieterse) [inline docs/faq.md]: should be the shortest of all??

File: docs/faq.md [hunk 2]
```diff
@@ -81,3 +81,14 @@ sudo systemctl disable cloud4rpi.service
...
+
+!!! Note
+    The variable update interval does not affect commands reception. Variable changes made using the Web UI (**Switch** or **Slider** widget) are applied instantly.
```
Comment (dirk-pieterse) [inline docs/faq.md]: commands reception?

**Review by dirk-pieterse — APPROVED** [review]

File: docs/faq.md (hunk 1 above)
Comment (Himura2la) [reply docs/faq.md]: If one type of measurements performs every 5 minutes and the other is every minute, you should send data every minute.

File: docs/faq.md (hunk 2 above)
Comment (Himura2la) [reply docs/faq.md]: The Cloud sends commands to devices. Devices receive these commands. And the update interval on devices does not affect this

File: docs/faq.md [hunk 3]
```diff
@@ -21,35 +21,35 @@ sudo pip install cloud4rpi
 
 ## Raspberry Pi does not recognize my 1-Wire device
 
-If you experience issues with the 1-Wire interface on Raspberry Pi, follow the instructions below.
+Follow the instructions below if you experience issues with the 1-Wire interface on Raspberry Pi.
 
 1. Run `sudo raspi-config` and make sure that the 1-Wire interface is enabled (**Interfacing Options** | **1-Wire** menu).
 2. Add the `dtoverlay=w1-gpio` string to your `/boot/config.txt` file (`echo dtoverlay=w1-gpio | sudo tee -a /boot/config.txt`).
 3. Reboot (`sudo reboot` command).
-4. Double-check the wiring and your device's operational voltage. The 1-Wire **DATA** bus should be connected to **GPIO4** (Pin 7) and pulled up to **VCC**.
```
Comment (Himura2la) [inline docs/faq.md:29]: That's incorrect. "Pulling up" means "connecting a pin to a positive voltage through a resistor" https://en.wikipedia.org/wiki/Pull-up_resistor

File: docs/faq.md (hunk 1 above)
Comment (dirk-pieterse) [reply docs/faq.md]: 'the shortest' means the shortest.

File: docs/faq.md (hunk 2 above)
Comment (dirk-pieterse) [reply docs/faq.md]: The variable update interval does not affect commands, and variable changes made through the Web UI (**Switch** or **Slider** widget) are applied instantly.

**Review by dirk-pieterse — APPROVED** [review]

File: docs/faq.md (hunk 1 above)
Comment (Himura2la) [reply docs/faq.md]: I don't see what's wrong. Short delay, shorter delay, the shortest delay. Is there some more appropriate word to specify small amount of time?

File: docs/faq.md [hunk 4]
```diff
@@ -81,3 +81,13 @@ sudo systemctl disable cloud4rpi.service
 sudo rm /lib/systemd/system/cloud4rpi.service
 sudo systemctl daemon-reload
 ```
+
+
+## How to choose the DATA_SENDING_INTERVAL value?
+
+You should call [device.publish_data()](/api/python/#publish_data) and [device.publish_diag()](/api/python/#publish_diag) functions frequently to update the monitored variables. The update frequency depends on the variable dynamics, for example, atmospheric conditions do not change every minute. This means the delay between updates for a weather station should be one to ten minutes.
+
+If you monitor several variables with different dynamics, the delay between updates should be the shortest.
```
Comment (dirk-pieterse) [inline docs/faq.md:90]: I can see you do not understand. 
1. "the delay between updates" Which delay'?
2. "should be the shortest of all" 'the largest', 'the most', 'the shortest', etc. do not require 'of all', and in this case, it isn't even clear what 'of all' refers to. What are you trying to say here?

**Pull request merged**